spot-client/src/common/remote-control/xmpp-connection.js

```diff
@@ -7,6 +7,15 @@ import { getJitterDelay } from 'common/utils';
 
 import { IQ_NAMESPACES, IQ_TIMEOUT } from './constants';
 
+/**
+ * How long will the join MUC function retry on conflict error before giving up.
+ * The value must be bigger than the idle client connection timeout. It's currently set to 30 seconds for the worst case
+ * scenario.
+ *
+ * @type {number}
+ */
+const CONFLICT_TIMEOUT = 35000;
```

Comment:
Is there access to the idle client connection timeout? Maybe this value can be set by adding to that idle client value?

Reply:
No, this value is defined in the prosody plugin in another repo.